fix(modal): removed drop shadow, updated colors

mattnolting opened the pull request:
fixes https://docs.google.com/spreadsheets/d/1fBCPPb4Z1sqUDKgQkpueGe8VQZo4pBUA9PV4vnFCnDw/edit#gid=0&range=A5

spadgett reviewed (commented):
/assign @rhamilto @sg00dwin 
/ok-to-test

sg00dwin commented on frontend/public/components/modals/_modals.scss:
```diff
@@ -35,29 +40,23 @@
   min-height: 100%;
   // --pf-c-modal-box--Padding* will not render correctly here so applying pf4 spacer values
   padding: 0 var(--pf-global--spacer--xl) var(--pf-global--spacer--xl);
-  @include scroll-shadows-vertical-covers;
```
Delete corresponding `@mixin` file 
frontend/public/style/mixin/_scroll-shadows.scss

sg00dwin replied:
Delete corresponding `@mixin` file 
frontend/public/style/mixin/_scroll-shadows.scss

sg00dwin commented on frontend/public/components/modals/_modals.scss:
```diff
@@ -19,14 +29,9 @@
 }
 
-.modal-body-border {
-  border-top: 1px solid $pf-color-black-300;
```
The original operator install modal designs included a line between the heading and scrolling body.
https://github.com/itsptk/openshift-origin-design/blob/3cbbabfc06819bea6eadb05dc85e49d218ad2eab/designs/administrator/olm/select-install-operator-version/index.md

mattnolting replied:
@sg00dwin updated

sg00dwin commented:
A couple of comments. Overall looks good.

sg00dwin commented on frontend/public/components/modals/_modals.scss:
```diff
@@ -35,29 +44,23 @@
   // so that input and textarea don't mask the inner scroll shadows
   input, textarea {
     &.pf-c-form-control {
-      --pf-c-form-control--BackgroundColor: transparent;
-
       max-width: 100%;
 
       &[disabled],
       &[readonly] {
-        --pf-c-form-control--BackgroundColor: rgba(128, 128, 128, 0.15);
+        --pf-c-form-control--BackgroundColor: var(--pf-global--disabled-color--300);
       }
     }
   }
 }
```
Lines 49 thru 60 should be deleted, they only supported scroll-shadows.

yanpzhan commented:
@mattnolting could you help list some related page to check for the update? Thanks

mattnolting commented:
@yanpzhan the `.modal-body-inner-shadow-covers` element contained `padding`, [removing it](https://github.com/openshift/console/pull/11148#discussion_r823986483) also removes the padding. To correct I've added the padding to `.modal-body-content`

![Screen Shot 2022-03-15 at 9 47 57 AM](https://user-images.githubusercontent.com/5385435/158391807-29a0a5d5-cadc-499b-9269-6bbb5c364889.png)

![Screen Shot 2022-03-15 at 9 41 04 AM](https://user-images.githubusercontent.com/5385435/158391835-37f2d8dd-93eb-4ce2-b12a-345b0e6dda61.png)

![Screen Shot 2022-03-15 at 9 40 48 AM](https://user-images.githubusercontent.com/5385435/158391854-affa7a81-9657-4182-a7bc-d4ee0f284b90.png)

![Screen Shot 2022-03-15 at 9 39 11 AM](https://user-images.githubusercontent.com/5385435/158391866-b4153348-247d-460b-bdd3-4ca787cf04a6.png)

spadgett reviewed (approved):
/approve
/label docs-approved
/label px-approved

We have docs- and px-approval for the epic.

yanpzhan commented:
Tested against the pr, the modal box has border issue, texts in all modals are too near to the left and right side of modal.
@mattnolting could you have a check?
![Screenshot from 2022-03-22 10-13-21](https://user-images.githubusercontent.com/7313189/159410008-90d676c7-3c05-47ae-9754-dfa1a3a0e604.png)
![Screenshot from 2022-03-22 10-13-47](https://user-images.githubusercontent.com/7313189/159410677-ad239211-e7a4-4dd9-8913-3517d333dcf8.png)
![Screenshot from 2022-03-22 10-11-46](https://user-images.githubusercontent.com/7313189/159410731-f68ac908-cde9-4d79-a255-c42574da52b5.png)

mattnolting commented:
@yanpzhan Thanks for looking at this! Here's what I'm seeing:

![Screen Shot 2022-03-22 at 2 50 05 PM](https://user-images.githubusercontent.com/5385435/159554488-7eb1781e-1678-4e68-83b5-91b86e27b0b5.png)

![Screen Shot 2022-03-22 at 2 48 41 PM](https://user-images.githubusercontent.com/5385435/159554489-ead3aa65-72ca-4327-86ea-097d7d8c88df.png)

![Screen Shot 2022-03-22 at 2 48 31 PM](https://user-images.githubusercontent.com/5385435/159554493-5aa536b9-8198-4751-9dcf-2f284d258e18.png)

![Screen Shot 2022-03-22 at 2 41 51 PM](https://user-images.githubusercontent.com/5385435/159554494-475c741f-6526-4d27-a57d-e467e2cd2158.png)

I rebased, updated and force pushed. Hopefully that will resolve the issues you're seeing.

yanpzhan commented:
Retested the pr after the codes are rebased, now all modals work well in dark theme.
/label qe-approved

vikram-raj commented:
/test frontend

rhamilto commented:
/retest

rhamilto commented:
@mattnolting, please squash your commits.

@sg00dwin, PTAnotherL

jcaianirh commented:
/retest

rhamilto commented:
Unrelated to the changes here, but we also have spacing issues in modals (example below).  Are these issues being tracked/fixed somewhere?

<img width="633" alt="Screen Shot 2022-03-30 at 11 36 16 AM" src="https://user-images.githubusercontent.com/895728/160874309-7b2fdb2c-3fac-4516-92cb-c1d99031f47c.png">

mattnolting commented:
> Unrelated to the changes here, but we also have spacing issues in modals (example below). Are these issues being tracked/fixed somewhere?

@rhamilto I don't believe so. I'd be happy to address in a followup!

sg00dwin commented:
/lgtm

rhamilto commented:
> Unrelated to the changes here, but we also have spacing issues in modals (example below). Are these issues being tracked/fixed somewhere?
> 
> <img alt="Screen Shot 2022-03-30 at 11 36 16 AM" width="633" src="https://user-images.githubusercontent.com/895728/160874309-7b2fdb2c-3fac-4516-92cb-c1d99031f47c.png">

Another instance I noticed:

<img width="639" alt="Screen Shot 2022-04-12 at 8 49 42 AM" src="https://user-images.githubusercontent.com/895728/162966121-e2c39b48-093c-455f-923b-41a50927f792.png">
